.eslintcache

```diff
@@ -1 +1 @@
-[{"/home/pawel/Projects/pasnik/libs/slack-api/jest.config.js":"1"},{"size":360,"mtime":1630329669693,"results":"2","hashOfConfig":"3"},{"filePath":"4","messages":"5","errorCount":0,"warningCount":0,"fixableErrorCount":0,"fixableWarningCount":0},"1hozt75","/home/pawel/Projects/pasnik/libs/slack-api/jest.config.js",[]]
\ No newline at end of file
+[{"C:\\Projekty\\React\\pasnik\\apps\\frontend\\src\\app\\CreateOrder\\CreateOrder.tsx":"1"},{"size":3997,"mtime":1632721098689,"results":"2","hashOfConfig":"3"},{"filePath":"4","messages":"5","errorCount":0,"warningCount":0,"fixableErrorCount":0,"fixableWarningCount":0},"9b7oed","C:\\Projekty\\React\\pasnik\\apps\\frontend\\src\\app\\CreateOrder\\CreateOrder.tsx",[]]
```

Comment:
what is this? 👀

Reply:
It is autogenerated by eslint, should go to gitignore

Reply:
@jakzda pls add this file to `.gitignore`